modules/evm-bridge/src/lib.rs

```diff
@@ -172,7 +171,11 @@ impl<T: Config> EVMBridgeTrait<AccountIdOf<T>, BalanceOf<T>> for Pallet<T> {
 			ExecutionMode::Execute,
 		)?;
 
-		Self::handle_exit_reason(info.exit_reason)
+		Self::handle_exit_reason(info.exit_reason)?;
+
+		// In order to avoid empty contracts, must check for return values
+		ensure!(info.output.len() == 32, Error::<T>::InvalidReturnValue);
```

Comment:
actually I think we should do something similar to SafeERC20 https://github.com/OpenZeppelin/openzeppelin-contracts/blob/165e6f19489786c9c6abfcc9bdc8f2815d807935/contracts/token/ERC20/utils/SafeERC20.sol#L66

It is fine to have no return, but if there is return, it should be true

Reply:
https://github.com/AcalaNetwork/Acala/pull/938/files#diff-7efb0e44e274d6054a16c345d5e935867bf0b44dc8960e5219e1a605aac72aa3R194

In order to support `balance_of(Non-existent address) = 0`, the `ExitSucceed::Stopped` have been added. But this is also the case when the contract is empty, so it must not be empty.

Reply:
ok I think it is fine to require output, but can you also require output is true